### DISCUSSION TITLE
Audiogridder server port issue

### DISCUSSION THREAD

**Discussion by valmax93** [apohl79/audiogridder#1256] [Help] [answered]
Good afternoon everybody. how can I fix this? Please help me (I hope you'll see the picture)

<img width="1680" alt="Screenshot 2023-08-01 alle 13 49 07" src="https://github.com/apohl79/audiogridder/assets/141143412/68dcd7c6-3672-48c7-a7d2-0d14333c6ef4">

**Comment by apohl79** [comment]
Ok that looks wrong. Could you provide the server log? Thx!

**Reply by valmax93** [reply]
Where can I find it?
Ps: I use server and plugin on the same device (Apple M1 MacOs Ventura latest release)

**Reply by valmax93** [reply]
I only would like to delete all the audiogridder server infos and install it from the very beginning if I can’t repeir this error

**Reply by apohl79** [reply]
https://audiogridder.com/bug-report-diagnostics-locations/

If you wipe the directory ~/.audiogridder, you will reset both plugins and server.

**Reply by valmax93** [reply]
Ok I’ll send what you linked me asap; I’ll even try to wipe that directory

**Reply by valmax93** [reply]
When I searched for audiogridder directory, I found nothing 😔

**Reply by apohl79** [reply]
It is hidden. Try `cmd + shift + .` in finder.

**Reply by valmax93** [reply]
Thanks!!!! I’ll try it asap and I’ll send you what you asked me

**Reply by valmax93** [reply]
I can’t find the folder even if I tried the shortcut you wrote me 😔

**Reply by valmax93** [reply]
I cannot find even the other folder you asked for

**Reply by valmax93** [reply]
Is there anything more I should try?

**Reply by apohl79** [reply, marked as answer]
https://github.com/apohl79/audiogridder/discussions/1173#discussioncomment-5257888

This resets your server settings. Run this in the terminal app.

**Reply by valmax93** [reply]
It just worked well!!! Thaks a lot, you saved my work!!!
